data/projects/nested_languages.md

```diff
@@ -0,0 +1,28 @@
+Multiple programming languages can coexist in a single source file. coala would
+benefit from having an API to support such analysis algorithms. If we have e.g.
```

Comment:
API to support analysis algorithms as in "the analysis algorithms are using the API"?
wouldn't it make more sense to strip the file apart beforehand and pass the different parts to different analysis routines?